### PR TITLE
PMM-12902 PMM 3.0.0 Alpha Rel Notes

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

File: docs/release-notes/3.0.0_Alpha.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+### More options for deploying PMM rootless
+
+In addition to Podman, PMM Server can also now be deployed rootless with Helm, Docker, Virtual Appliance, or Amazon AWS.
+
+This setup prioritizes security by eliminating the need for root privileges to create, run, and manage containers. Running PMM Server as a non-root user means that you never have to give a user root permission on the host. This adds an extra layer of security, essential for protecting against potential security breaches.
+
+For detailed instructions on deploying rootless PMM, check the [Setting up PMM Server](https://docs.percona.com/percona-monitoring-and-management/setting-up/index.html#set-up-pmm-server) topic.
+
+### Transition to Oracle Linux 9 completed
```
Comment (rnovikovP) [inline docs/release-notes/3.0.0_Alpha.md:39]: maybe shorten this section to just say - RHEL 7 is dead , so no EL 7 builds, everything on OL9 - as most secured and recent version of OS

File: docs/release-notes/3.0.0_Alpha.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+With the release of PMM 3, we are completing the transition away from CentOS/RHEL 7-based builds and fully embracing Oracle Linux 9 as the default operating system for PMM-base images. This transition began with PMM 2.38 and is now being finalized for all users, including those who had upgraded from 2.37 and earlier versions using the **PMM Upgrade** option on the Home Dashboard.
+
+By leveraging Oracle Linux 9 as the default operating system, we are ensuring a more stable, updatable and well-maintained operating system for PMM's base images.
+
+## Improved UX
+
+### Grafana upgraded to latest version
```
Comment (rnovikovP) [inline docs/release-notes/3.0.0_Alpha.md:47]: Let's move this section to the bottom of the highlights as this is inherited features 

(I'll additionally review them )

File: docs/release-notes/3.0.0_Alpha.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+![Administration menu](../_images/Admin_menu.png)
+
+For more details about all the changes coming Grafana v10.4, see [Grafana's release blog](https://grafana.com/blog/2024/03/06/grafana-10.4-release-all-the-latest-features/).
+
+## Enhanced PMM deployment on AMI and OVF environments
+
+PMM is now deployed inside containers within AMI and OVF instances. These containers operate in a rootless mode and are managed via Podman. This approach significantly increases security for a more robust and secure monitoring of your environments.
+
+## Finalized DBaaS migration to Percona Everest
```
Comment (rnovikovP) [inline docs/release-notes/3.0.0_Alpha.md:76]: let's add here about seamless integration from everest to PMM when you use pmm for monitoring (this  should be already in latest version of Everest )

File: docs/release-notes/3.0.0_Alpha.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+All PMM-related configuration and inventory options have been relocated from the **Configuration** section to a unified **Administration** menu.
+Grouping all administration-related tools means that you get easier, centralized access to all PMM configuration settings, making it easier to find what you need.
+
+![Administration menu](../_images/Admin_menu.png)
+
+For more details about all the changes coming Grafana v10.4, see [Grafana's release blog](https://grafana.com/blog/2024/03/06/grafana-10.4-release-all-the-latest-features/).
+
+## Enhanced PMM deployment on AMI and OVF environments
+
+PMM is now deployed inside containers within AMI and OVF instances. These containers operate in a rootless mode and are managed via Podman. This approach significantly increases security for a more robust and secure monitoring of your environments.
```
Comment (BupycHuk) [inline docs/release-notes/3.0.0_Alpha.md:74]: and provides the same experience across different deployment types

**Review by BupycHuk — APPROVED** [review]

**Review by Nailya — APPROVED** [review]

File: docs/release-notes/3.0.0_Alpha.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+- [PMM-12545](https://perconadev.atlassian.net/browse/12545): Finished transferring DBaaS features to Percona Everest, a standalone solution for easy database deployment and management, seamlessly integrated with PMM for database monitoring.
+
+- [PMM-12940](https://perconadev.atlassian.net/browse/PMM-12940): Implemented Watchtower which automatically replaces the PMM Docker Server container, simplifying upgrades for Docker, Podman, and OVF/AMI setups.
+
+- [PMM-12523](https://perconadev.atlassian.net/browse/PMM-12523): PMM Server can now be deployed rootless using Podman, Helm, Docker, Virtual Appliance, or Amazon AWS, enhancing security by eliminating the need for root privileges.
+
+- [PMM-12526](https://perconadev.atlassian.net/browse/PMM-12526): Replaced API keys with Grafana service accounts for enhanced API authentication, fine-grained access control, and improved logging and security.
+
+- [PMM-8306](https://perconadev.atlassian.net/browse/PMM-8306): Deployed PMM inside rootless containers managed by Podman within AMI and OVF instances, enhancing security and providing a consistent experience across different deployment types.
```
Comment (ademidoff) [inline docs/release-notes/3.0.0_Alpha.md:133]:
```suggestion
- [PMM-8306](https://perconadev.atlassian.net/browse/PMM-8306): Deployed PMM as a rootless container managed by Podman within AMI and OVF instances, enhancing security and providing a consistent experience across different deployment types.
```

Comment (ademidoff) [reply docs/release-notes/3.0.0_Alpha.md:133]: This seems to be more correct.

**Review by ademidoff — CHANGES_REQUESTED** [review]

File: docs/release-notes/3.0.0_Alpha.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+Leveraging the new Grafana user interface updates, we've taken the opportunity to refine the workflow for creating alerts from the **Alert Rules** and **Alert Rule Templates** pages.
+
+You'll notice separate, more visible options for creating different types of alert rules, cutting down on unnecessary steps and making it easier to manage various alert rules, templates, and configurations:
+
+![New Alerts rule page](../_images/Grafana_Alerting.png)
+
+### Simplified Administration Settings with Dedicated Menus
+
+We've made administration settings easier to manage by introducing a new **Administration** menu. This menu brings together all Grafana-related configurations and account management options that were previously scattered across the general **Configuration** section. This change helps administrators locate and manage Grafana-specific settings more efficiently.
```
Comment (matejkubinec) [inline docs/release-notes/3.0.0_Alpha.md:76]: Just a note, the "Administration" menu (Merged "Configuratio" and "Server Admin") was introduced by Grafana.

**Pull request merged**